lib/_http_server.js

```diff
@@ -211,8 +212,8 @@ function writeHead(statusCode, reason, obj) {
     }
     if (k === undefined) {
       if (this._header) {
-        throw new Error('Can\'t render headers after they are sent to the ' +
-                        'client');
+        throw new errors.Error('ERR_ASSERTION', 'Can\'t render headers ' +
```

Comment:
`ERR_ASSERTION` is the wrong choice here. That should be reserved for uses of the `assert` module. A new error code would be more appropriate.